components/NavBar/NavBar.tsx

```diff
@@ -24,59 +18,14 @@ import {
 import { FormattedMessage, useIntl } from "react-intl"
 import styled from "styled-components"
 
-import { Container } from "../Container"
-import { Link } from "../Link"
-
-import { Cart } from "./Cart"
-
-const UserNavbarContainer = styled(Container)`
-  width: 100%;
-  background-color: ${tokens.colors.ui.background__default.hex};
-  display: flex;
-  justify-content: flex-end;
-`
-
-const UserNavbar = styled.nav`
-  display: flex;
-  align-items: center;
-  padding: 0.5rem 0;
-  margin-bottom: 1rem;
-`
-
-const Header = styled.header`
-  background-color: ${tokens.colors.ui.background__default.hex};
-  width: 100%;
-  padding: 0.5rem 0;
-  position: sticky;
-  top: 0;
-  border-bottom: 2px solid ${tokens.colors.ui.background__light.hex};
+const EDSTopBar = styled(TopBar)`
   margin-bottom: 5rem;
+  margin-left: 8rem;
```

Comment:
The Top bar would look better if it has 100% width, so we should remove these side margins.  With these margins added, the border doesn't look good and we get a strange behaviour while scrolling
![Skjermbilde 2022-08-22 kl  08 26 45](https://user-images.githubusercontent.com/1568252/185853972-1657e7d7-1aeb-4a58-85e9-a4692f2c7e68.png)

Reply:
Updated